inference-engine/tests/functional/plugin/shared/src/single_layer_tests/prior_box_clustered.cpp

```diff
@@ -1,185 +0,0 @@
-            float center_y = (h + offset) * stepH;
-
-            for (size_t s = 0; s < numPriors; ++s) {
-                float box_width = widths[s];
-                float box_height = heights[s];
-
-                float xmin = (center_x - box_width / 2.0f) / imgWidth;
-                float ymin = (center_y - box_height / 2.0f) / imgHeight;
-                float xmax = (center_x + box_width / 2.0f) / imgWidth;
-                float ymax = (center_y + box_height / 2.0f) / imgHeight;
-
-                if (clip) {
-                    xmin = (std::min)((std::max)(xmin, 0.0f), 1.0f);
-                    ymin = (std::min)((std::max)(ymin, 0.0f), 1.0f);
-                    xmax = (std::min)((std::max)(xmax, 0.0f), 1.0f);
-                    ymax = (std::min)((std::max)(ymax, 0.0f), 1.0f);
-                }
-
-                topData_0[h * layerWidth * numPriors * 4 + w * numPriors * 4 + s * 4 + 0] = xmin;
-                topData_0[h * layerWidth * numPriors * 4 + w * numPriors * 4 + s * 4 + 1] = ymin;
-                topData_0[h * layerWidth * numPriors * 4 + w * numPriors * 4 + s * 4 + 2] = xmax;
-                topData_0[h * layerWidth * numPriors * 4 + w * numPriors * 4 + s * 4 + 3] = ymax;
-
-                for (int j = 0; j < varSize; j++)
-                    topData_1[h * layerWidth * numPriors * varSize + w * numPriors * varSize +
-                    s * varSize +
-                    j] = variances[j];
-            }
-        }
-    }
-
-    // Be aligned with test utils ref calulcation method, which returns std::vector<std::vector<uint8_t>>...
-    std::vector<std::vector<uint8_t>> ret(1);
-    for (auto& val : outBuf) {
-        uint8_t* u8_val = reinterpret_cast<uint8_t*>(&val);
-        ret[0].push_back(u8_val[0]);
-        ret[0].push_back(u8_val[1]);
-        ret[0].push_back(u8_val[2]);
-        ret[0].push_back(u8_val[3]);
-    }
-
-    return ret;
-}
-
-void PriorBoxClusteredLayerTest::SetUp() {
-    priorBoxClusteredSpecificParams specParams;
-    std::tie(specParams, netPrecision,
-        inputShapes, imageShapes, targetDevice) = GetParam();
-
-    std::tie(widths,
-        heights,
-        clip,
-        step_width,
-        step_height,
-        offset,
-        variances) = specParams;
-
-    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-    auto paramsIn = ngraph::builder::makeParams(ngPrc, { inputShapes, imageShapes });
-    auto paramsOut = ngraph::helpers::convert2OutputVector(
-        ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(paramsIn));
-
-    ngraph::op::PriorBoxClusteredAttrs attributes;
-    attributes.widths = widths;
-    attributes.heights = heights;
-    attributes.clip = clip;
-    attributes.step_widths = step_width;
-    attributes.step_heights = step_height;
-    attributes.offset = offset;
-    attributes.variances = variances;
-
-    auto priorBoxClustered = std::make_shared<ngraph::op::PriorBoxClusteredIE>(
```

Comment:
Sorry if I'm out of the context, but why the test was deleted at all? As far as I understand, the test has to be modified to use `opset3::PriorBoxClustered` instead of `PriorBoxClusteredIE` and rely on calling interpreter backend or constant folding (both will relly on `evaluate` I guess).

Reply:
I tried to use `opset3::PriorBoxClustered` instead of `PriorBoxClusteredIE`, but faced an issue related to inference or conversion when the output of the network is a constant, in progress.

Reply:
@mikhail-treskin  Unfortunately, looks like plugins don't support `Parameter-> (const folded subgraph) -> Result` cases. Also, I even can't implement a workaround for this case, e.g.  `Result <- Parameter-> (const folded subgraph) -> Result` or another due to issues in plugins.  I'll create a JIRA bug and restore the tests in disabled mode + ticket to enable these tests